composer.json

```diff
@@ -18,6 +20,13 @@
             "Keboola\\ManageApiTest\\": "tests/"
         }
     },
+    "scripts": {
+        "ci": [
+            "@phpcs"
+        ],
+        "phpcs": "phpcs -s -n -p --extensions=php . src tests",
+        "tests": "phpunit"
```

Comment:
Přidány standardně testy, aby bylo jasné jak fungují.

Reply:
tady ty "tests" znamenají ale vlastně "app"

Reply:
no "app" tady úplně není. Je to knihovna - klient. To je jakoby "app". A to, že využíváme testy klienta na něco dalšího je druhá věc.